pkg/config/config.go

```diff
@@ -1123,6 +1123,17 @@ type Config struct {
 	// Default: empty
 	// Public: No
 	SelfInstrumentationTelemetryEndpoint string `yaml:"self_instrumentation_telemetry_endpoint" envconfig:"self_instrumentation_telemetry_endpoint"`
+
+	// Ntp is a map for ntp configuration. It is enabled by default and it uses public ntp server pool https://www.ntppool.org/en/use.html
```

Comment:
Do we have the default NTP for this feature in the end?

Reply:
nope, good catch 👍 updating